portable/IAR/AtmelSAM7S64/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
+/** \fn    AT91F_PMC_EnablePCK */
+/** \brief Enable peripheral clock */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PMC_EnablePCK( AT91PS_PMC pPMC,  /* \arg pointer to PMC controller */
+                                       unsigned int pck, /* \arg Peripheral clock identifier 0 .. 7 */
+                                       unsigned int mode )
+    {
+        pPMC->PMC_PCKR[ pck ] = mode;
+        pPMC->PMC_SCER = ( 1 << pck ) << 8;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_DisablePCK */
+/** \brief Enable peripheral clock */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PMC_DisablePCK( AT91PS_PMC pPMC,   /* \arg pointer to PMC controller */
+                                        unsigned int pck ) /* \arg Peripheral clock identifier 0 .. 7 */
+    {
+        pPMC->PMC_SCDR = ( 1 << pck ) << 8;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_EnableIt */
+/** \brief Enable PMC interrupt */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PMC_EnableIt( AT91PS_PMC pPMC,    /* pointer to a PMC controller */
+                                      unsigned int flag ) /* IT to be enabled */
+    {
+        /** Write to the IER register */
+        pPMC->PMC_IER = flag;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_DisableIt */
+/** \brief Disable PMC interrupt */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PMC_DisableIt( AT91PS_PMC pPMC,    /* pointer to a PMC controller */
+                                       unsigned int flag ) /* IT to be disabled */
+    {
+        /** Write to the IDR register */
+        pPMC->PMC_IDR = flag;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_GetStatus */
+/** \brief Return PMC Interrupt Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PMC_GetStatus( /* \return PMC Interrupt Status */
+        AT91PS_PMC pPMC )                      /* pointer to a PMC controller */
+    {
+        return pPMC->PMC_SR;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PMC_GetInterruptMaskStatus */
+/** \brief Return PMC Interrupt Mask Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PMC_GetInterruptMaskStatus( /* \return PMC Interrupt Mask Status */
+        AT91PS_PMC pPMC )                                   /* pointer to a PMC controller */
```

Comment:
Should this be one line?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)